tests/setData.spec.ts

```diff
@@ -123,4 +123,45 @@ describe('setData', () => {
       'Cannot add property count'
     )
   })
+
+  it('https://github.com/vuejs/vue-test-utils-next/issues/538', async () => {
```

Comment:
super nit: I prefer when we have a readable test name instead of just a link to a Github issue (easier to understand when the test is failing)

Reply:
Sure, I'll update